packages/client/.storybook/tennisClubMembershipCertifiedCertificateTemplate.ts

```diff
@@ -14,7 +14,7 @@ export const tennisClubMembershipCertifiedCertificateTemplate = `
   <g clip-path="url(#clip0_5_2)">
     <path d="M583 0H0V842H583V0Z" fill="white" />
     <text fill="#222222" xml:space="preserve" style="white-space: pre" font-family="Noto Sans" font-size="8" letter-spacing="0em">
-      <tspan x="62" y="426.352">{{ $lookup $metadata "modifiedAt"}}</tspan>
+      <tspan x="62" y="426.352">{{ $lookup ($action "PRINT_CERTIFICATE") "createdAt" }}</tspan>
```

Comment:
After this change, this will be the officially supported way to get the `Date of certification` to a certificate. FYI @tumbledwyer @Siyasanga

Reply:
or not, as we can't filter on the action values yet

Reply:
I recommend using a custom handlebar helper in country config for these sort of use cases so we can keep the core API as generic as we can

Reply:
Agreed, if we expose the `content` property from the current handlebars, it should be fairly simple to create a custom handlebar on countryconfig for `copiesPrinted`